packages/components/ng-ui-router-title/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@ovh-ux/ng-ui-router-title",
+  "version": "2.0.0",
+  "description": "AngularJS module for updating browser title/history based on the current ui-router state.",
+  "keywords": [
+    "angular",
+    "ovh",
+    "title",
+    "uirouter"
+  ],
+  "repository": "ovh-ux/ng-ui-router-title",
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/ng-ui-router-title.js",
+  "scripts": {
+    "build": "rollup -c --environment BUILD:production",
+    "dev": "rollup -c --environment BUILD:development",
+    "dev:watch": "yarn run dev --watch",
+    "prepare": "yarn run build",
+    "start": "lerna exec --stream --scope='@ovh-ux/ng-ui-router-title' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/ng-ui-router-title' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/ng-ui-router-title' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "dependencies": {
+    "lodash": "^4.17.15"
+  },
+  "devDependencies": {
+    "@ovh-ux/component-rollup-config": "7.0.0"
+  },
+  "peerDependencies": {
+    "@uirouter/angularjs": "^1.0.0",
+    "angular": "1.6.10"
```

```suggestion
    "angular": "^1.7.5"
```
or at least 
```suggestion
    "angular": "^1.6.10"
```
to match with `angular` given version